docs/ui/insight/get-started.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Get started with RedisInsight
+linkTitle: Get started
+description: Get started with RedisInsight
+weight: 40
+
+---
+
+RedisInsight is available as a standalone download or [bundled with Redis Stack](/docs/stack/get-started/install). Install it and try the in-app examples available in Workbench.
+
+
+## Install standalone RedisInsight 
+
+You can use RedisInsight on MacOS, Linux, or Windows.
+
+**Install on MacOS**
+
+Redis offers RedisInsight on Intel and M1. 
+
+1. Download [RedisInsinght](https://redis.com/redis-enterprise/redis-insight). 
+
+1. Add RedisInsight to **Applications**. 
+
+1. From your Terminal, run this command.
+
+```
+redisinsight
+``` 
+
+To resume your journey, [start Redis Stack Server](#start-redis-stack-server).   
+
+**Install on Linux**
```

Comment:
I believe these sections are wrong also looping in @ViktarStarastsenka 
and it should be fixed across all OS platforms
1. via the application store
2. via a package manager
3. installing manually

you can't say `For appImage` you also don't say `For .exe` or `.app`